rules/possible-errors/function.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
     // Disallow an unspaced operator within `calc` functions
     'function-calc-no-unspaced-operator': true,
     // Disallow direction values in `linear-gradient()` calls that are not valid according to the standard syntax
-    'function-linear-gradient-no-nonstandard-direction': true
+    'function-linear-gradient-no-nonstandard-direction': true,
+    // Disallow unknown functions
+    'function-no-unknown': true
```

Comment:
The package.json peer needs to be updated to 14.4.0. Please don't forget to merge this PR as either a `!` scope or "BREAKING CHANGE" body (or both).

Reply:
@jzempel could you help me understand when a peer needs to be updated in this project? Also how is the version determined (e.g. `^14.4.0` vs `^14.5.0`)?

Reply:
without it, stylelint will fail when it encounters the new `function-no-unknown` rule [added in v14.4.0](https://github.com/stylelint/stylelint/blob/HEAD/CHANGELOG.md#1440).